scalameter-core/src/main/scala/org/scalameter/Context.scala

```diff
@@ -9,16 +9,17 @@ import org.scalameter.Key._
 
 
 case class Context(properties: immutable.Map[Key[_], Any]) {
+  def -[T](t: Key[T]) = Context(properties - t)
   def +[T](t: (Key[T], T)) = Context(properties + t)
   def ++(that: Context) = Context(this.properties ++ that.properties)
   def ++(that: Seq[KeyValue]) = Context(this.properties ++ that)
   def get[T](key: Key[T]) = properties.get(key).asInstanceOf[Option[T]].orElse {
```

Comment:
Please add the return type here, since it's not immediately apparent from the rhs expression.